Vagrantfile

```diff
@@ -98,6 +98,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     backend.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-backend-federated-cluster.pp"
   end
 
+  # Unable to use at this time
+  # https://github.com/dotless-de/vagrant-vbguest/issues/367
```

Comment:
You use TODO elsewhere in the code comments. Should probably have here, too so it is easy to find these.

Reply:
TODO added